### ISSUE TITLE
Mouse Click + Panning not being recognized in a certain condition

### ISSUE THREAD

**Issue by bakermanbrian** [EpicGames/PixelStreamingInfrastructure#166]
## Component your question relates to
- [ ] Frontend library

## Question
I've run into an issue in my pixel streaming project where when I access a server on mobile, then access the same server on Desktop, that, until I click a UMG button in my game, my mouse click + panning isn't being recognized by my game. It works fine on Desktop, it then works fine on the mobile touch, but it stops working when I then went from mobile to Desktop. I am using Flags.MatchViewportResolution.

**Comment by lukehb** [comment]
What versions of everything are you running?

UE version?
PS Infra version?

**Comment by bakermanbrian** [comment]
5.4.2 and 5.4 branch

**Comment by lukehb** [comment]
They are both streaming at the same time, or one, then you close it, then open the other one? I am having trouble reproducing it, could you send a minimum example UE project?

**Comment by bakermanbrian** [comment]
Hey @lukehb it is the same stream, accessed from Mobile then Desktop (so close the mobile connection, then open the Desktop). This is with screen resizing turned on. I can see if I can get a minimum UE project with the issue I can share.

**Comment by lukehb** [comment]
Thanks a repro project would be much appreciated. Is the screen resizing option required to reproduce the issue?

**Comment by bakermanbrian** [comment]
Hey @lukehb believe it is. I can share a project, is there a private way I could do that? Feel free to email me brian at archilabs dot ai and I can send it there.

**Comment by lukehb** [comment]
Sent you an email!

**Comment by lukehb** [comment]
@MWillWallT When you get a chance can you share your repro attempt here (I passed you a link to this project)?

**Comment by MWillWallT** [comment]
Hi there! Thank you for your patience

So I've attempted to reproduce the issue a number of times using the provided project. To summarise the reproduction steps:

Connect via mobile with Match viewport enabled
Play around, then disconnect
Connect to stream via desktop
Mouse click and panning should not work

However I found that on every connection from desktop, there was no issue with mouse input, both click and pan were working as intended. I've made sure to match the UE and infra versions as reported.

IF we could get a video demonstration or more information that would be lovely
